doc/source/whatsnew/v0.13.0.txt

```diff
@@ -790,7 +790,7 @@ Experimental
   .. ipython:: python
 
      for o in pd.read_msgpack('foo.msg',iterator=True):
-        print o
```

Comment:
I would revert this one.   I see no reason to update really OLD documentation that we don't use anymore.